<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeckWindow</class>
 <widget class="QDialog" name="DeckWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListView" name="listViewDeck">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>341</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditQuestion">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>440</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>SizeFDiagCursor</cursorShape>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditAnswer">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>440</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>130</y>
     <width>129</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButtonStudy">
      <property name="text">
       <string>Study Deck</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonNew">
      <property name="text">
       <string>Add Flashcard</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonUpdate">
      <property name="text">
       <string>Update Flashcard</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonDelete">
      <property name="text">
       <string>Delete Flashcard</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonNext">
      <property name="text">
       <string>Next Flashcard</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonPrevious">
      <property name="text">
       <string>Previous Flashcard</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonMenu">
      <property name="text">
       <string>Return to Main Menu</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="questionLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>420</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Question:</string>
   </property>
  </widget>
  <widget class="QLabel" name="answerLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>420</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
